skycultures/greek_leidenAratea/description.en.utf8

```diff
@@ -92,7 +92,7 @@ Both, the Almagest and the poem, are written in Greek originally but the Leiden
 	<tr valign="top">
 		<td><notr>Delphinus, Del</notr></td>
 		<td><notr>&Delta;&epsilon;&lambda;&phi;&#943;&sigmaf;</notr></td>
-		<td>Dolphin <br /> In the Greek original (Aratos, ~300 BCE), there was no constellation Equuleus. The Farnese Globe (as well as the Mainz Globe and the Kugel Globe) depicts the Dolphin rahter largely, covering the space of Equ. That is why, I follow this Aratean tradition also for the Leiden Aratea.</td>
+		<td>Dolphin <br /> In the Greek original (Aratos, ~300 BCE), there was no constellation Equuleus. The Farnese Globe (as well as the Mainz Globe and the Kugel Globe) depicts the Dolphin rahter largely, covering the space of Equ. That is why I follow this Aratean tradition also for the Leiden Aratea.</td>
```

Comment:
another typo: rather

Reply:
That is why I **rather** follow this Aratean tradition **comma** also for ...

Reply:
oh, yes, delete comma

Reply:
@sushoff "rather" was not inserted after all, correct? but I now also see another typo, "rahter".

I think the phrase should read
> ... depicts the Dolphin **rather** largely, covering the space of Equ. That is why I [rather?] follow this Aratean tradition >>,<< also for the Leiden Aratea...

or maybe even better
> ... depicts the Dolphin **rather** largely, covering the space of Equ. That is why I **chose to** follow this Aratean tradition for the Leiden Aratea...

Reply:
yes